### DISCUSSION TITLE
Visits api

### DISCUSSION THREAD

**Discussion by RakeshPotnuru** [KendallDoesCoding/KendallDoesCoding#7] [Ideas]
[countapi](https://countapi.xyz)

**Comment by KendallDoesCoding** [comment]
Hi, following our discussion in your chat section on your website, I need help with the visits button my personal portfolio website too.

Portfolio Code: https://github.com/KendallDoesCoding/portfolio

I want the button around the same place as it is on your website, in the footer in the top right or left corner. 

Thanks a lot for your help,
KendlalDoesCoding

**Comment by RakeshPotnuru** [comment]
Run this in your browser 
```
https://api.countapi.xyz/create?namespace=kendalldoescoding.gq&enable_reset=1
```
copy the key and store it in `.env` file

**Reply by KendallDoesCoding** [reply]
Where's the .env file?

**Reply by KendallDoesCoding** [reply]
{"namespace":"kendalldoescoding.gq","key":"607bb42f-5bd1-4c4d-b8d1-b3cf6c4ebe4e","value":0}

**Comment by RakeshPotnuru** [comment]
Is this okay?
<img width="675" alt="image" src="https://user-images.githubusercontent.com/70439799/152770969-7f3b7768-b323-4458-9a49-ec3df5cb1304.png">

**Reply by KendallDoesCoding** [reply]
yup, that works too but maybe change the design of it, to something like on your website.

**Reply by RakeshPotnuru** [reply]
I added countapi logic, so you can customize yourself

**Reply by KendallDoesCoding** [reply]
It seems good as it is in your PR, but how do I customize it? There's only a script tag added by you.

**Reply by RakeshPotnuru** [reply]
Just change the CSS
Customize this tag 👇
```
<p>Visits: <span id="visits">0</span></p>
```

**Comment by KendallDoesCoding** [comment]
Ok, will do
